debian/mariadb-server-10.6.postinst

```diff
@@ -238,9 +238,9 @@ EOF
   ;;
 
   triggered)
-    if [ -x "$(command -v systemctl)" ]; then
-      systemctl daemon-reload
-    else
+    if [ -d /run/systemd/system ]; then
+      systemctl --system daemon-reload
+    elif [ -x "/etc/init.d/mariadb" ]; then
```

Comment:
harmless, but you don't need to quote paths without spaces.

Reply:
Agree but I guess that @elboulangero just re-used [l.272](https://github.com/MariaDB/server/pull/1787/files/3e7a54eeb25c2973fbbf736282d5209134fe87e5#diff-cbec972052c4cfdfcfc528132339f80ccea59df512df6c54b27ac8eb31d89a1dR272)

Reply:
Correct @fauust. Let me push an update without the double quotes.

Reply:
@elboulangero, may I ask you to modify also l272 for consistency :)
Thanks !